src/Command/Redis/XACKDEL.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2025 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis;
+
+use Predis\Command\PrefixableCommand as RedisCommand;
+
+/**
+ * @see http://redis.io/commands/xackdel
+ */
+class XACKDEL extends RedisCommand
```

Comment:
You need to update `ClientInterface` and `ClientContextInterface` files to reflect new commands. This is were our public API defined